src/client/pages/forum/forms/PostForm.jsx

```diff
@@ -0,0 +1,83 @@
+import Tag from "../Tag";
+import {useState} from "react";
+import Category from "../Category";
+
+const PostForm = (props) => {
+	const { posts, setPosts } = props;
+	const [tags, setTags] = useState([]);
+	const [categories, setCategories] = useState([]);
+	const [newPost, setNewPost] = useState({
+		title: '',
+		content: '',
+	});
+
+	const changeHandler = (e) => {
+		const name = e.target.name;
+        const value = e.target.value;
+        setNewPost({...newPost, [name]: value});
+
+		if(name === 'Gaming' || name === 'Fan-Fiction' || name === 'Cosplay' || name === 'Manga' || name === 'Tv/Film'){
```

Comment:
Instead of checking for all the valid values inside this if condition maybe you should check if the name/value is not the default one, since it's either the default one or one of these five.

Reply:
I changed the category from select HTML to input, because I wanted to have an array, so I changed my category .